tests/test_integration/test_code_generation.py

```diff
@@ -156,9 +156,6 @@ def stencil(field_in: gtscript.Field[np.float_], field_out: gtscript.Field[np.fl
 
 @pytest.mark.parametrize("backend", ALL_BACKENDS)
 def test_lower_dimensional_inputs(backend):
-    if backend == "gtc:cuda":
-        pytest.xfail("gtc:cuda backend does not support lower dimensional fields")
-
```

Comment:
This test seems to be in the wrong file, no? It's a full test, not only code generation. Not your fault, though...

Reply:
Yes, you are right. Should probably live in `tests/test_integration/test_suites.py`or so…